enclave-runtime/src/initialization/parentchain/parachain.rs

```diff
@@ -63,12 +61,12 @@ impl FullParachainHandler {
 
 		let genesis_header = params.genesis_header.clone();
 
-		let light_client_seal = EnclaveLightClientSeal::new(base_path.join(LIGHT_CLIENT_DB_PATH))?;
+		let light_client_seal = GLOBAL_LIGHT_CLIENT_SEAL.get()?;
```

Comment:
It is now initialized at an earlier stage and made available as a global component.